src/framework/components/anim/controller.js

```diff
@@ -20,67 +20,74 @@ import {
  * @param {number|pc.Vec2} point - The coordinate/vector thats used to determine the weight of this node when it's part of a pc.BlendTree.
  * @param {number} speed - The speed that it's pc.AnimTrack should play at.
  */
-function AnimNode(state, parent, name, point, speed) {
-    this._state = state;
-    this._parent = parent;
-    this._name = name;
-    this._point = Array.isArray(point) ? new pc.Vec2(point) : point;
-    this._speed = speed || 1.0;
-    this._weight = 1.0;
-    this._animTrack = null;
-}
+class AnimNode {
+    constructor(state, parent, name, point, speed) {
+        this._state = state;
+        this._parent = parent;
+        this._name = name;
+        this._point = Array.isArray(point) ? new pc.Vec2(point) : point;
+        this._speed = speed || 1.0;
```

Comment:
Make default parameter. Also, JSDoc above it should be updated to specify `speed` is optional, defaulting to 1.

Reply:
There's also a doc bug for point - it's an array of numbers.

Reply:
Fixed.